connector/kafka2ftp/test/unit/ftpFtpClientPool.test.js

```diff
@@ -0,0 +1,125 @@
+const FTPClientPool = require('../../app/ftp/FTPClientPool');
+const FTPClient = require('../../app/ftp/FTPClient');
+
+jest.mock('../../app/ftp/FTPClient');
+jest.mock('../../app/Utils');
+
+
+const ftpConfig = {
+  host: 'myftpserver.com',
```

Comment:
In the Qualcomm's case, the user and password is defined by CIMCAMP or they are using a default credentials? I think that is good they uses a strong credentials for a security connection.

Reply:
I think it is defined by CIMCAMP or it is a closed network, but in any case it can be set by environment variable.